apps/etterlatte-behandling/src/main/kotlin/behandling/generellbehandling/GenerellBehandlingDao.kt

```diff
@@ -7,45 +7,53 @@ import no.nav.etterlatte.libs.common.objectMapper
 import no.nav.etterlatte.libs.common.tidspunkt.getTidspunkt
 import no.nav.etterlatte.libs.common.tidspunkt.setTidspunkt
 import no.nav.etterlatte.libs.database.setJsonb
+import no.nav.etterlatte.libs.database.single
 import no.nav.etterlatte.libs.database.singleOrNull
 import no.nav.etterlatte.libs.database.toList
 import java.sql.Connection
 import java.sql.ResultSet
 import java.util.UUID
 
 class GenerellBehandlingDao(private val connection: () -> Connection) {
-    fun opprettGenerellbehandling(generellBehandling: GenerellBehandling) {
-        with(connection()) {
+    companion object {
+        const val DATABASE_FELTER = "id, innhold, sak_id, opprettet, type, tilknyttetBehandling"
+    }
+
+    fun opprettGenerellbehandling(generellBehandling: GenerellBehandling): GenerellBehandling {
+        return with(connection()) {
             val statement =
                 prepareStatement(
                     """
-                    INSERT INTO generellbehandling(id, innhold, sak_id, opprettet, type)
-                    VALUES(?::UUID, ?, ?, ?, ?)
+                    INSERT INTO generellbehandling($DATABASE_FELTER)
```

Comment:
Misliker streng-interpolering i sql prepared statements ganske sterkt. Hadde foretrukket manuell repetering evt slå sammen create og patch i en statement heller hvis du vil kutte ned på repetisjon

Reply:
Synes ikkje vi har klart å finne eit veldig bra mønster for SQL i koden vår generelt. Prøvde meg på ein ny variant i [migrering](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2202/files#diff-629b8f7e9b5643765f502d9dd2c0a2cbc3be4006e8790c52f98cc81eac9b301b) som iallfall tvinger fram at vi er litt konsekvente.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2277/commits/859ce1d9407bc0cddcebedd7e9caf36532554ab4

Reply:
Synes det er forskjell på string interpolering som er ekstern og fra en statisk lokalt definert variabel